File: ontrack-web-core/components/widgets/home/ValidationMetricsChartWidget.js
import ValidationStampMetricsChart from "@components/validationStamps/ValidationStampMetricsChart";
import ValidationChartWidget from "@components/widgets/home/ValidationChartWidget";

export default function ValidationMetricsChartWidget(props) {
    return (
        <ValidationChartWidget
            prefix="Metrics of"
            chart={ValidationStampMetricsChart}
            {...props}
        />
    )
}

File: ontrack-model/src/main/java/net/nemerosa/ontrack/model/dashboards/widgets/ValidationMetricsChartWidget.kt
package net.nemerosa.ontrack.model.dashboards.widgets

import org.springframework.stereotype.Component

@Component
class ValidationMetricsChartWidget : AbstractWidget<ValidationChartWidgetConfig>(
    key = "home/ValidationMetricsChart",
    name = "Validation metrics",
    description = "Chart displaying metrics of a validation over time.",
    defaultConfig = ValidationChartWidgetConfig(project = null, branch = null, validationStamp = null),
    preferredHeight = 30,
)
